spec/Conticket/Model/Aggregates/Event/TicketLifespanSpec.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace spec\Conticket\Model\Aggregates\Event;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Conticket\Model\Aggregates\Event\TicketEndDateMustBeGreaterThanStartDateException;
+
+class TicketLifespanSpec extends ObjectBehavior
+{
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Conticket\Model\Aggregates\Event\TicketLifespan');
+    }
+
+    function it_throws_an_exception_when_start_date_is_greater_than_end_date()
+    {
+        $this->beConstructedThrough('fromStartAndEnd', ['2016-01-01', '2015-01-01']);
+        $this->shouldThrow(TicketEndDateMustBeGreaterThanStartDateException::class);
+    }
+
+    function it_should_return_datetimes_objects_on_start_and_end_methods()
```

Comment:
Those class from PhpSpec returns something in background, I'm getting error defining the return type to void